config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -54,3 +54,6 @@ spec:
         volume1:
           containerImage: quay.io/tripleozedcentos9/openstack-cinder-volume:current-tripleo
           replicas: 1
+  nova:
+    template:
+      secret: osp-secret
```

Comment:
Nova defaults is set in a way that this sample will deploy:
* nova-api
* cell0 with nova-conductor
* (eventually nova-scheduler (in progress))

This is enough to show that the control plane works. I.e services are up, controlplane only operations can be done, e.g. flavor create, keypair create.

The nova-operator also supports multiple cells computes in other than cell0 with
```
  secret: osp-secret
  cellTemplates:
    cell0:
      cellDatabaseUser: nova_cell0
      cellMessageBusInstance: rabbitmq
      hasAPIAccess: true
    cell1:
      cellDatabaseUser: nova_cell1
      cellMessageBusInstance: rabbitmq-cell1
      hasAPIAccess: true
```
But please note that cell0 and cell1 needs separate RabbitMQCluster instance (i.e. rabbitmq vs rabbitmq-cell1) otherwise nova-computes will not communicate properly.